modules/simple-payments/paypal-express-checkout.js

```diff
@@ -145,12 +145,22 @@ var PaypalExpressCheckout = {
 				return new paypal.Promise( function( resolve, reject ) {
 					jQuery.post( PaypalExpressCheckout.getCreatePaymentEndpoint( blogId ), payload )
 						.done( function( paymentResponse ) {
+							if ( ! paymentResponse ) {
+								PaypalExpressCheckout.showError( PaypalExpressCheckout.processErrorMessage(), domId );
```

Comment:
This will cause an error :(
`PaypalExpressCheckout.processErrorMessage` currently expects the first argument to be an object: https://github.com/Automattic/jetpack/pull/7525/files#diff-ad5f66f7ce71d416f68444397b6e59cdR91

Just need to update processErrorMessage. This should work `var error = errorResponse && errorResponse.responseJSON;`